lib/membrane_rtc_engine/tees/EngineTee.ex

```diff
@@ -0,0 +1,118 @@
+defmodule Membrane.RTC.Engine.Tee do
+  @moduledoc """
+  Element for forwarding buffers to at least one output pad
+
+  It has one input pad `:input` and 2 output pads:
+  * `:master` - is a static pad which is always available and works in pull mode
+  * `:copy` - is a dynamic pad that can be linked to any number of elements (including 0) and works in push mode
+
+  The `:master` pad dictates the speed of processing data and any element (or elements) connected to `:copy` pad
+  will receive the same data as `:master`.
+
+  It has got built-in mechanism for limiting forwarding video buffers.
+  It reads from ETS table on which pads it should forward buffers.
+  """
+
+  use Membrane.Filter
+
+  def_options track_id: [
+                spec: String.t(),
+                description: "Id of track for which tee was created"
+              ],
+              ets_name: [
+                spec: String.t(),
+                description:
+                  "Name of ETS table from which Tee will read to which pads it should send buffers",
+                default: "table"
+              ],
+              type: [
+                spec: :audio | :video,
+                description: "Type of track which buffers tee is forwarding"
+              ]
+
+  def_input_pad :input,
+    availability: :always,
+    mode: :pull,
+    demand_unit: :buffers,
+    caps: :any
+
+  def_output_pad :master,
+    availability: :always,
+    mode: :pull,
+    caps: :any
+
+  def_output_pad :copy,
+    availability: :on_request,
+    mode: :push,
+    caps: :any
+
+  @impl true
+  def handle_init(opts) do
+    {:ok,
+     %{
+       ets_name: :"#{opts.ets_name}",
+       track_id: opts.track_id,
+       counter: 0,
```

Comment:
Do we need this counter when we receive message when track priorities have changed?

Reply:
We need it for the SRTP rollover counters

Reply:
I don't get this but we can discuss this tomorrow